src/waveform/renderers/waveformrenderbeat.cpp

```diff
@@ -22,9 +22,9 @@ WaveformRenderBeat::~WaveformRenderBeat() {
 void WaveformRenderBeat::setup(const QDomNode& node, const SkinContext& context) {
     m_beatColor.setNamedColor(context.selectString(node, "BeatColor"));
     m_beatColor = WSkinColor::getCorrectColor(m_beatColor).toRgb();
-
     if (m_beatColor.alphaF() > 0.99)
         m_beatColor.setAlphaF(0.9);
```

Comment:
This gets overridden later. 
Should we still allow to set an beat grid alpha from the skins? 
... probably not.

Reply:
No, this should be configurable by the user independently of the skin.